Index: src/main/java/module-info.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>module org.example.rewardmanagementsystem {\r\n    requires javafx.controls;\r\n    requires javafx.fxml;\r\n    requires javafx.web;\r\n\r\n    requires org.controlsfx.controls;\r\n    requires com.dlsc.formsfx;\r\n    requires net.synedra.validatorfx;\r\n    requires org.kordamp.ikonli.javafx;\r\n    requires org.kordamp.bootstrapfx.core;\r\n    requires eu.hansolo.tilesfx;\r\n    requires com.almasb.fxgl.all;\r\n    requires java.sql;\r\n\r\n    opens org.example.rewardmanagementsystem to javafx.fxml;\r\n    exports org.example.rewardmanagementsystem;\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/module-info.java b/src/main/java/module-info.java
--- a/src/main/java/module-info.java	(revision dd6d840a081611f2210a4f06dd046d2db1f1c8de)
+++ b/src/main/java/module-info.java	(date 1734330338600)
@@ -11,6 +11,7 @@
     requires eu.hansolo.tilesfx;
     requires com.almasb.fxgl.all;
     requires java.sql;
+    requires mysql.connector.j;
 
     opens org.example.rewardmanagementsystem to javafx.fxml;
     exports org.example.rewardmanagementsystem;
